src/components/MaterialUI/DataDisplay/PredefinedElements/ActionModal.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import Modal from "./../Modal";
+import ModalProps from "./../modalProps";
+import Button from "@material-ui/core/Button";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import sharedMessages from "../../../../sharedMessages";
+import { FormattedMessage } from "react-intl";
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Don't need to style de Modals if you are using the default orc-shared one. All is defined there
talking about message here